### ISSUE TITLE
Filter freq CV control

### ISSUE THREAD

**Issue by PawelHadrian** [Miserlou/RJModules#28]
First of all - I love your modules :) But they need some tweaking ;)
Filter freq CV control just doesn't work as I expect it to work and is a real pain in the ass if you want to control it with external midi controller or LFOs. It looks like the knob on 12 o'clock (both LP and HP filters off) is at +10V. I think that it should be (two options): 
1. bi-polar with -12V (fully counter clockwise) and +12V (fully clockwise) with 0V being 12 o'clock.
2. uni-polar with 0V (fully counter clockwise) and +12V (fully clockwise) with +6V being 12 o'clock.

**Comment by Miserlou** [comment]
Yep. I didn't/still pretty much don't really know what I was doing w/r/t CV.

This should be fixed - how are you controlling RJModules with a MIDI controller? That sounds awesome?

**Comment by PawelHadrian** [comment]
MIDI CV core module and a novation LaunchControl. VCVRack is just perfect for live knob tweaking :)
